tests/unit/conftest.py

```diff
@@ -137,13 +138,19 @@ def pyramid_request(db_session, application_instance, lti_v11_params):
 
 
 @pytest.fixture
-def user_is_learner(pyramid_request):
-    pyramid_request.lti_user.roles = "Learner"
+def user_is_learner(pyramid_request, lti_user):
+    lti_user.lti_roles = [
+        factories.LTIRole(scope=RoleScope.COURSE, type=RoleType.LEARNER)
+    ]
+    pyramid_request.lti_user = lti_user
```

Comment:
Is this last line necessary? I think `pyramid_request.lti_user` is already `lti_user`

Reply:
Nope, not necessary :+1: